Example/Build-V3-Plugin-Monitoring-ios/AppCoordinator.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  AppCoordinator.swift
+//  FASDKBuild-Plugin-Optimizers-ios
+//
+//  Created by Jérôme Morissard on 05/17/2022.
+//  Copyright (c) 2022 Jérôme Morissard. All rights reserved.
+//
+
+import Build_V3_Plugin_Monitoring_ios
+import FASDKBuild_ios
+import Foundation
+import WebKit
+
+class AppCoordinator: FABaseAppCoordinator, FirebasePerformanceDelegate {
```

Comment:
Can we just make FirebasePerformanceDelegate an extension inside this file and move the proper methods there?